rules-tests/Php85/Rector/MethodCall/NullDebugInfoReturnRector/Fixture/normal_function.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Php85\Rector\MethodCall\NullDebugInfoReturnRector\Fixture;
+
+function __debugInfo(): ?array
```

Comment:
this detected as `Function_` node, not `ClassMethod`, so not included in test coverage. Create under a class.